Can't compile because of missing dependency

Hello, first of all: thanks for the project!

I'm trying to experiment with it, but can't compile since Bintray Repo is no longer available (https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/) and one of the dependencies uses it:  `com.palantir.docker.compose:docker-compose-rule-core:jar:0.34.0` .

```shell
[ERROR] Failed to execute goal on project kafka-connect-solr: Could not resolve dependencies for project com.github.jcustenborder.kafka.connect:kafka-connect-solr:jar:0.1-SNAPSHOT: Could not find artifact com.palantir.docker.compose:docker-compose-rule-core:jar:0.34.0
```

Recent versions (>=1.0.0) are  available in Maven Central: https://search.maven.org/artifact/com.palantir.docker.compose/docker-compose-rule-core .

--- a/pom.xml
+++ b/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <groupId>com.github.jcustenborder.kafka.connect</groupId>
         <artifactId>kafka-connect-parent</artifactId>
-        <version>2.4.0</version>
+        <version>2.8.0-1</version>
     </parent>
     <artifactId>kafka-connect-solr</artifactId>
     <version>0.1-SNAPSHOT</version>

